src/main/java/frc/robot/subsystems/OuttakeSubsystem.java

```diff
@@ -41,12 +42,15 @@ public OuttakeSubsystem() {
     m_shooterMotorI.follow(m_shooterMotorII, true);
     m_shooterMotorI.setIdleMode(IdleMode.kBrake);
     m_shooterMotorII.setIdleMode(IdleMode.kBrake);
-
-    m_shooterMotorI.setSmartCurrentLimit(90);
-    m_shooterMotorII.setSmartCurrentLimit(90);
-    m_takeNoteMotor.setSmartCurrentLimit(90);
+    m_pivotMotor.setSmartCurrentLimit(OUTTAKE_CURRENT_LIMITS);
```

Comment:
The pivot motor should be getting a different current limit. This means it should be using a different constant.